### DISCUSSION TITLE
R studio repository  publishing Quarto website_ Error

### DISCUSSION THREAD

**Discussion by Manikandanramaraj2008** [community/community#129873] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi,
I was tried to publish my R studio quarto website in branch main.... the below error message recvd. Request pls support me  to solve this. i
404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.

[GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

**Comment by LiteBrite82** [comment]
I'm not familiar with Quarto; did you follow the [publishing guide](https://quarto.org/docs/publishing/github-pages.html)? 
It looks like it didn't get "published"

**Reply by Manikandanramaraj2008** [reply]
Hi,
Thanks  for replying.  yes, i have followed... Thanks for the effort and support
manikandan

**Reply by MustafaCode0** [reply]
Have you tried adding index.html to your repo? Your website should be in the index file.

**Reply by nicole-torno** [reply]
I recently had a similar experience using quarto to publish a dashboard hosted on github pages. Through much trial and error, I discovered that the required syntax to publish your update has changed. Using the publish command for gh-pages setup, originally the syntax was "quarto publish gh-pages". However, now it seems that the document must be named explicitly for the publish command to be able to locate and use it. I have found that simply adding the file name at the end of the command, as in "quarto publish gh-pages index.qmd" works consistently now. I hope this helps, and I hope they update the quarto documentation soon to reflect this change
